views/assets/src/tailwind.css

```
@@ -1,744 +1,773 @@
+  border-collapse: collapse;
+  margin: 0.75em 0;
+  table-layout: auto;
+}
+#wedevs-project-manager .prose table th,
+#wedevs-project-manager .prose table td,
+#wedevs-project-manager .ProseMirror table th,
+#wedevs-project-manager .ProseMirror table td {
+  border: 1px solid hsl(var(--border));
+  padding: 6px 10px;
+  vertical-align: top;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)